tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php

```diff
@@ -319,4 +319,83 @@ public function testListFloatTypeColumns()
         $this->assertFalse($columns['col']->getUnsigned());
         $this->assertTrue($columns['col_unsigned']->getUnsigned());
     }
+
+    /**
+     * Test default null values (especially for mariadb 10.2.7+)
+     * @link https://github.com/doctrine/dbal/issues/2817
+     */
+    public function testNullColumnDefaultWithTable()
```

Comment:
Would it make sense to move this test one level of abstraction above and run for any platform?

Reply:
Mmmh, maybe. Need to improve a bit before.

Reply:
` : void`